### PR TITLE
Charlie Station

### PR DESCRIPTION
**Pull request by Carolyn3114**
## About the PR
A (currently) admin-only spawnable separate station.
Set place on "Charlie Station", a long lost station that predates Space Station 13 and 14, with four survivors waking up from cryosleep, finding their once state-of-the-art research facility has gone into disarray with damage, overgrowth, and fauna.

## Why / Balance
They *do* get a magnet and a circuit imprinter, not sure if this is too much.

## Media
I was too lazy to do maprender
image

- [ ] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes

**Changelog**

### CODE REVIEW

**Comment by Colin-Tel** [comment]
How is a user supposed to spawn in as an OldStationMedic or whatnot? are there spawners for these things, or is an admin just supposed to give them the outfits?

**Comment by Carolyn3114** [comment]
> How is a user supposed to spawn in as an OldStationMedic or whatnot? are there spawners for these things, or is an admin just supposed to give them the outfits?

uses ghost role spawner

File: Resources/Prototypes/Roles/Jobs/Fun/misc_startinggear.yml
Comment (DebugOk) [inline Resources/Prototypes/Roles/Jobs/Fun/misc_startinggear.yml:1]: Improper seperation, this shouldn't be in base files

**Review by DebugOk — CHANGES_REQUESTED** [review]

File: Resources/Prototypes/Roles/Jobs/Fun/misc_startinggear.yml
Comment (Carolyn3114) [reply Resources/Prototypes/Roles/Jobs/Fun/misc_startinggear.yml:1]: oops

Comment (Carolyn3114) [reply Resources/Prototypes/Roles/Jobs/Fun/misc_startinggear.yml:1]: fixed

**Review by DebugOk — APPROVED** [review]
Fuck it we ball

**Comment by Carolyn3114** [comment]
hip hurray

**Pull request merged**